common/src/main/java/org/opensearch/ml/common/utils/StringUtils.java

```diff
@@ -293,4 +294,45 @@ public static String getJsonPath(String jsonPathWithSource) {
         // Extract the substring from the startIndex to the end of the input string
         return (startIndex != -1) ? jsonPathWithSource.substring(startIndex) : jsonPathWithSource;
     }
+
+    /**
+     * Checks if the given input string matches the dot path format.
```

Comment:
are we matching dot path format or json path format? Description and method names both are different to each other.

Reply:
right, json path is more straight forward. I will change to  json path format